plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -165,8 +165,7 @@ export class ExpressInstrumentation extends InstrumentationBase<
       ) {
         const route = original.apply(this, args);
         const layer = this._router.stack[this._router.stack.length - 1];
-        instrumentation._applyPatch.call(
-          instrumentation,
+        instrumentation._applyPatch(
```

Comment:
Note for reviewer: this usage of `Function.protptype.call` comes from this commit https://github.com/open-telemetry/opentelemetry-js-contrib/pull/355/commits/0723d0666b914e132962744b6454c736e5bd165e as a new addition.

I did not dig further to get the original implementation but I think there is no need to use `.call(...)` since we have access to the container object and therefore calling `_applyPatch` with the right context.